libs/membership/api/src/lib/dashboard/dashboard.module.ts

```diff
@@ -0,0 +1,17 @@
+import {Module} from '@nestjs/common'
+import {PrismaModule} from '@wepublish/nest-modules'
+import {DashboardInvoiceResolver} from './invoices/dashboard-invoice.resolver'
+import {DashboardInvoiceService} from './invoices/dashboard-invoice.service'
+import {DashboardSubscriptionResolver} from './subscriptions/dashboard-subscription.resolver'
+import {DashboardSubscriptionService} from './subscriptions/dashboard-subscription.service'
+
```

Comment:
The path of this file is endless. Could we simplify? We have now lib in lib folder.

Reply:
Paths shouldnt matter for us. A clear path is better and recommended by nest itself.